app/controllers/concerns/public_token_magic_links_management.rb

```diff
@@ -30,4 +42,12 @@ def handle_error!(error_type)
     )
     redirect_to login_path
   end
+
+  def access_token
```

Comment:
```suggestion
  def magic_token_show_params
```

c'est plus conventionnel comme ça